src/assets/user-info-settings/index.css

```
@@ -0,0 +1,298 @@
+html {
+	background: #FF0000;
+}
+
+body {
+	font-family: Poppins, Arial, Helvetica, sans-serif;
+	font-size: 15px;
+	width: 1280px;
+	margin: 0;
+	overflow: auto;
+	box-sizing: border-box;
+	background: #EAEAEA;
+}
+
+button:active,
+.server-selection input:checked + label {
+	box-shadow: inset 0 0px 10px 0 rgba(66, 45, 120, 0.75) !important;
+	color: #9D6FF3;
+}
+
+header {
+	position: relative;
+	background: #37A985;
+	color: #FFF;
+	height: 70px;
+	line-height: 70px;
+	border-bottom: 2px solid #4F2E8C;
+	background: -webkit-gradient(linear, left top, left bottom, from(#9D6FF3), to(#673DB6));
+	z-index: 19;
+}
+
+header h1 {
+	margin: 0;
+	font-size: 30px;
+	text-align: center;
+	font-weight: normal;
+}
+
+button,
+.button,
+input[type="submit"] {
+	font-family: Poppins, Arial, Helvetica, sans-serif;
+	color: #45297A;
+	min-width: 200px;
+	box-sizing: content-box;
+	-webkit-box-sizing: content-box;
+	-webkit-box-align: center;
+	-webkit-box-pack: center;
+	min-height: 60px;
+	text-align: center;
+	font-size: 28px;
+	background: -webkit-gradient(linear, left top, left bottom, from(#FFF), color-stop(0.5, #FFF), color-stop(0.8, #F6F6F6), color-stop(0.95, #F5F5F5), to(#BBB))0 0;
+	border-radius: 12px;
+	cursor: pointer;
+	border: 0;
+	box-shadow: 0 1px 5px rgba(0, 0, 0, 0.4);
+	margin: 5px;
+}
+
+.body-content {
+	margin: 30px 60px 0 200px;
+}
+
+div.group h2 {
+	font-weight: normal;
+}
+
+div.group {
+	width: 485px;
+	display: inline-block;
+	margin: 0 5px;
+}
+
+div.body-content > h1 {
+	margin: 15px 0 0 0;
+}
+
+div.group > * {
+	margin: 5px 0;
+	font-size: 30px;
+}
+
+div.group > p.content,
+div.group > input[type="text"],
+div.group > select {
+	background: #FFF;
+	color: #6A6C75;
+	padding: 10px;
+	border-radius: 8px;
+	font-size: 25px;
+	box-shadow: none;
+	border: none;
+	width: 100%;
+	height: 58px;
+	box-sizing: border-box;
+	appearance: none;
+	-webkit-appearance: none;
+	-moz-appearance: none;
+	text-indent: 1px;
+	text-overflow: '';
+	line-height: 38px;
+	user-select: none;
+}
+
+div.group > input[type="text"] {
+	color: #000;
+}
+
+div.group > select {
+	content: ' ';
+	background-color: #FFF;
+	background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='32' height='32' fill='%23673DB6' viewBox='0 0 256 256'%3E%3Cpath d='M213.66,101.66l-80,80a8,8,0,0,1-11.32,0l-80-80A8,8,0,0,1,53.66,90.34L128,164.69l74.34-74.35a8,8,0,0,1,11.32,11.32Z'%3E%3C/path%3E%3C/svg%3E");
+	background-size: 40px;
+	background-position: 440px center;
+	background-repeat: no-repeat;
+	color: #9D6FF3;
+	cursor: pointer;
+}
+
+
+div.group > span {
+	color: #000;
+  }
+  div.group > svg {
```

Nitpick. Spaces here

```suggestion
}
div.group > svg {
```